build.gradle

```diff
@@ -333,7 +346,7 @@ task checkOutGhPages {
     if (!new File('tmp_gh-pages').exists()) {
```

Comment:
Since at the end of this script we now do removal of tmp_gh-pages (`rm -r tmp_gh-pages`), please consider failing the execution of the script if `!new File('tmp_gh-pages').exists() == false`. It should be logical, since if tmp_gh-pages already exists, then it probably contains some custom content, which should not be automatically removed; or the project directory structure is in incorrect state and should be cleaned up before running the scrip (`!new File('tmp_gh-pages').exists() == true` is practically an invariant for this script). It may cause failures on the first execution of the script on a new machine, which previously released gax, but it should be ok, if the error message is descriptive enough.